texttospeech/cloud-client/audio_profile.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python
+
+# Copyright 2018 Google Inc. All Rights Reserved.
```

Comment:
Google LLC

Reply:
Good point, but it is also Google Inc in every other file.

Reply:
For new files, I've been told we make the effort to update them, but on older files we just leave them.

Reply:
Done